must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -64,3 +64,20 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+kenel_collection() {
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
```

Comment:
adding `--to-namespace` throws an error saying `unknown flag`. But I've seen it everywhere in the must-gather code. What am I missing?.

Reply:
If by "everywhere" you mean in one other file, okay. :) Could be that you're using an old version of oc?

Reply:
That might be the reason, ill check my oc version.